### PR TITLE
Add ProxySQL documentation

### LINKED ISSUE #581: Add ProxySQL documentation
Following environment variables can be customized by the user:
- `CC_ENABLE_MYSQL_PROXYSQL`: `true` or `false`. Defaults to `false`
- `CC_MYSQL_PROXYSQL_MAX_CONNECTIONS`: Number. Defaults to `10`
- `CC_MYSQL_PROXYSQL_USE_TLS`: `true` or `false`. Deaults to `true`

Required environment variables: those exposed by a MySQL add-on

Added environment variable if the feature is enabled:
- `CC_MYSQL_PROXYSQL_SOCKET_PATH`: path to the unix socket to connect to the proxy

### PR DESCRIPTION
**Pull request by BlackYoup**
Fix #581

WIP because I didn't check for spelling mistakes and all yet. Let me know of the content / examples, if you would add anything or have any remarks :pray:

### CODE REVIEW

**Comment by gaelreyrol** [comment]
Nice ! It's clear and helpful !

I don't think it should be under the `administrate` folder so I wonder if it wouldn't be preferable to put this as a section in the MySQL add-on page and move usages section the dedicated pages, in `deploy/application/php/php-apps` and `deploy/application/php/tutorials/tutorial-wordpress`.

**Comment by gaelreyrol** [comment]
Also there is no section in the environment variable reference page !

**Comment by BlackYoup** [comment]
> I don't think it should be under the `administrate` folder so I wonder if it wouldn't be preferable to put this as a section in the MySQL add-on page and move usages section the dedicated pages, in `deploy/application/php/php-apps` and `deploy/application/php/tutorials/tutorial-wordpress`.

I feel like this would be better to have this as a separate documentation instead of having it in the MySQL documentation. But yes, we could reference it in the MySQL, Wordpress and PHP documentation pages.

But at the same time, I'm not sure where this could go indeed.

> Also there is no section in the environment variable reference page !

Yes, that's in my todo :)

**Comment by BlackYoup** [comment]
I made various changes, hopefully corrected the spelling mistakes, added the environment variables to the reference environment variables page. It should now be reviewable.

File: deploy/addon/mysql/proxysql.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: ProxySQL for MySQL add-ons
+position: 3
+shortdesc: How to configure and use ProxySQL for MySQL add-ons
+tags:
+- administrate
```
Comment (gaelreyrol) [inline deploy/addon/mysql/proxysql.md:6]: You might want to change this tag for `addons` ?
```suggestion
- addons
```

**Comment by BlackYoup** [comment]
I also forgot to add a section about metrics, will do..

**Comment by BlackYoup** [comment]
I added the metrics

**Comment by gaelreyrol** [comment]
LGTM :+1:

**Comment by BlackYoup** [comment]
I think I'll need to reference that page in the wordpress documentation too

**Comment by BlackYoup** [comment]
@gaelreyrol or maybe you could add proxysql when you update the PR about wordpress and bedrock? https://github.com/CleverCloud/doc.clever-cloud.com/pull/442 I'm not really sure how the environment variable are used

**Pull request merged**